### PR TITLE
Bug 2026668: Add "disconnected" annotation.

### PR DESCRIPTION
**Pull request by jsafrane**
The operator should work without connection to the internet. It needs only connection to AWS API + AWS EFS servers.

@openshift/storage

### CODE REVIEW

File: config/manifests/4.10/aws-efs-csi-driver-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -9,6 +9,7 @@ metadata:
     capabilities: Full Lifecycle
     containerImage: quay.io/openshift/origin-aws-efs-csi-driver-operator:latest
     support: Red Hat
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
```
Comment (bertinatto) [inline config/manifests/4.10/aws-efs-csi-driver-operator.clusterserviceversion.yaml]: I think we should also add `proxy-aware` to this list. What do you think? @jsafrane @duanwei33

Comment (bertinatto) [reply config/manifests/4.10/aws-efs-csi-driver-operator.clusterserviceversion.yaml]: Oh, and we definitely need to add `csi`! https://github.com/openshift/openshift-docs/blob/main/modules/osdk-csv-manual-annotations.adoc

Comment (jsafrane) [reply config/manifests/4.10/aws-efs-csi-driver-operator.clusterserviceversion.yaml]: I am not sure about the proxy. Sure, API calls to AWS can (and should) be proxied, however, the CSI driver / efs-utils directly mount NFS from somewhere inside AWS, which will not be proxied.

I added `"csi"`.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**